.eslintrc.js

```diff
@@ -1,6 +1,9 @@
 module.exports = {
     'extends': 'airbnb',
+    'rules': {
+      'react/jsx-filename-extension': [1, { "extensions": [".js", ".jsx"] }],
```

Comment:
Sorry, if you're saying we shouldn't have `.jsx` extensions, shouldn't we only be allowing `.js` extensions then?